### ISSUE TITLE
Dependabot not automatically merging for JavaScript projects

### ISSUE THREAD

**Issue by wilmol** [dependabot/dependabot-core#1983]
I have set up two projects with @dependabot, one is a Java project and the other a JavaScript project. I set up everything exactly the same (actually it uses the account settings - "Treat PR approval as a request to merge").

With the Java project, everything is working fine, but with the JavaScript project the PRs are not automatically being merged.

- Example PR on the Java project: https://github.com/wilmol/java-gradle-template/pull/12
- Example PR on the JavaScript project: https://github.com/wilmol/sudoku-solver-react.js/pull/7

Basically I expect @dependabot to behave the same in the JavaScript project as in the Java one.

**Comment by mattt** [comment]
Automatic Merging is a feature of Dependabot Preview, which was [deprecated in April](https://github.blog/2021-04-29-goodbye-dependabot-preview-hello-dependabot/) and [shut down on August 3rd](https://github.blog/changelog/2021-08-03-dependabot-preview-is-shutting-down/). There are no current plans to add this functionality to GitHub-native Dependabot. See: https://github.com/dependabot/dependabot-core/issues/1973#issuecomment-640918321
